src/util/worker/future.rs

```diff
@@ -75,11 +79,14 @@ where
     R: Runnable<T> + Send + 'static,
     T: Display + Send + 'static,
 {
+    let name = thread::current().name().unwrap().to_owned();
     let mut core = Core::new().unwrap();
     let handle = core.handle();
     {
         let f = rx.take_while(|t| Ok(t.is_some())).for_each(|t| {
             runner.run(t.unwrap(), &handle);
+            WORKER_PENDING_TASK_VEC.with_label_values(&[&name]).sub(1.0);
```

Comment:
seem we can introduce a count value, not only we can use gauge set here but also can use it to check is_busy.

Reply:
Maybe later, this PR only handles the metrics.